addons/account_payment/models/payment_transaction.py

```diff
@@ -89,14 +89,14 @@ def _compute_reference_prefix(self, provider_code, separator, **values):
                 return separator.join(invoices.mapped('name'))
         return super()._compute_reference_prefix(provider_code, separator, **values)
 
-    def _set_canceled(self, state_message=None):
+    def _set_canceled(self, state_message=None, extra_allowed_state=None):
         """ Update the transactions' state to 'cancel'.
 
         :param str state_message: The reason for which the transaction is set in 'cancel' state
         :return: updated transactions
         :rtype: `payment.transaction` recordset
         """
```

Comment:
The docstrings need to be adapted to reflect the change of signature

Reply:
Done :)